developers.diem.com/src/components/docs/CLI/index.js

```diff
@@ -1,12 +1,52 @@
-import React from 'react';
-import Terminal from 'diem-cli';
+import React from "react";
+import PropTypes from "prop-types";
+import Terminal, {
+  CommandForms as DefaultCommands,
+} from "@libra-opensource/diem-cli";
 
-import './styles.css';
-import classnames from 'classnames';
-import styles from './styles.module.css';
+import "./styles.css";
+import classnames from "classnames";
+import styles from "./styles.module.css";
 
-const CLI = () => {
-  return <Terminal className={classnames("cli", styles.root)} />;
+const getAvailableCommands = (command, commands) => {
+  if (command && commands) {
+    throw "command and commands can not be simultaneously set on a CLI";
+  } else if (!commands && !command) {
+    return DefaultCommands;
+  } else if (command) {
+    return [command];
+  } else {
+    return commands;
+  }
+};
+
+const CLI = ({ command, commands, isEmbedded, withTutorial }) => {
```

Comment:
Why would we have `command` and `commands`? Any way to simplify to `commands`?

Reply:
I have an explanation in the prop types, command is just a convenience prop. This is going to be commonly used with only one command